easybuild/easyconfigs/c/cURL/cURL-7.54.1-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'cURL'
+version = '7.54.1'
+
+homepage = 'http://curl.haxx.se'
+
+description = """
+ libcurl is a free and easy-to-use client-side URL transfer library,
+ supporting DICT, FILE, FTP, FTPS, Gopher, HTTP, HTTPS, IMAP, IMAPS, LDAP,
+ LDAPS, POP3, POP3S, RTMP, RTSP, SCP, SFTP, SMTP, SMTPS, Telnet and TFTP.
+ libcurl supports SSL certificates, HTTP POST, HTTP PUT, FTP uploading, HTTP
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = ['http://curl.haxx.se/download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+osdependencies = [
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+]
```

Comment:
@JackPerdue Same here, I would include the commented out `OpenSSL` dependency (together with a commented out `# configopts = "--with-ssl=$EBROOTOPENSSL"`) to make it clear this can also be provided through EasyBuild?